src/helpers/blockTime.js

```diff
@@ -0,0 +1,29 @@
+const MILLISECONDS_IN_A_SECOND = 1000
+
+export const blockTime = (eth) =>
+  async (blockNumber, showBlock = true, averageBlockTime = 13.965) => {
+    let timestamp
+    const currentBlock = await eth.getBlockNumber()
```

Comment:
Rather than doing both a `eth_blockNumber`, I would suggest just doing a `eth_getBlockByNumber` here, since that will contain both the block number and timestamp needed for later computation.